### PR TITLE
HOCS-6016: Track migration state

### PR DESCRIPTION
**Pull request by rramgolam**
Draft PR.

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/domain/migration/model/MigrationState.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    @Column(name="id")
+    private Long id;
+
+    @Column(name = "case_uuid")
+    protected UUID caseId;
+
+    @Column(name = "stage_uuid")
+    protected UUID stageId;
+
+    @Column(name = "cms_id")
```
Comment (SWhitlam) [inline src/main/java/uk/gov/digital/ho/hocs/domain/migration/model/MigrationState.java:30]: could this be called external_case_id

File: src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationFailureService.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package uk.gov.digital.ho.hocs.queue.migration;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class MigrationFailureService {
```
Comment (SWhitlam) [inline src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationFailureService.java:8]: I'm not sure this is needed as this data could be stored in the message_log table?

Comment (LiamMacP) [reply src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationFailureService.java:8]: For me the status in the message_log table should say failed, we should log out appropriately the error and then we can cross reference to remediate

**Comment by LiamMacP** [comment]
Closed due to work completed under #206

**Pull request closed without merging**